README.md

```diff
@@ -54,85 +72,70 @@ The Optimism Immunefi program offers up to $2,000,042 for in-scope critical vuln
 ├── <a href="./op-proposer">op-proposer</a>: L2-Output Submitter, submits proposals to L1
 ├── <a href="./op-service">op-service</a>: Common codebase utilities
+├── <a href="./op-ufm">op-ufm</a>: Simulations for monitoring end-to-end transaction latency
 ├── <a href="./op-wheel">op-wheel</a>: Database utilities
+├── <a href="./ops">ops</a>: Various operational packages
 ├── <a href="./ops-bedrock">ops-bedrock</a>: Bedrock devnet work
 ├── <a href="./packages">packages</a>
 │   ├── <a href="./packages/chain-mon">chain-mon</a>: Chain monitoring services
 │   ├── <a href="./packages/common-ts">common-ts</a>: Common tools for building apps in TypeScript
-│   ├── <a href="./packages/contracts-ts">contracts-ts</a>: ABI and Address constants
 │   ├── <a href="./packages/contracts-bedrock">contracts-bedrock</a>: Bedrock smart contracts
+│   ├── <a href="./packages/contracts-ts">contracts-ts</a>: ABI and Address constants
 │   ├── <a href="./packages/core-utils">core-utils</a>: Low-level utilities that make building Optimism easier
+│   ├── <a href="./packages/fee-estimation">fee-estimation</a>: Tools for estimating gas on OP chains
 │   └── <a href="./packages/sdk">sdk</a>: provides a set of tools for interacting with Optimism
+│   └── <a href="./packages/web3js-plugin">web3js-plugin</a>: Adds functions to estimate L1 and L2 gas
+├── <a href="./patches">patches</a>: Is this used somewhere or should it be deleted?
 ├── <a href="./proxyd">proxyd</a>: Configurable RPC request router and proxy
 └── <a href="./specs">specs</a>: Specs of the rollup starting at the Bedrock upgrade
+└── <a href="./ufm-test-services">ufm-test-services</a>: Runs a set of tasks to generate metrics
 </pre>
 
-## Branching Model
+## Development and Release Process
 
-### Active Branches
+### Overview
 
-| Branch          | Status                                                                           |
-| --------------- | -------------------------------------------------------------------------------- |
-| [master](https://github.com/ethereum-optimism/optimism/tree/master/)                   | Accepts PRs from `develop` when intending to deploy to production.                  |
-| [develop](https://github.com/ethereum-optimism/optimism/tree/develop/)                 | Accepts PRs that are compatible with `master` OR from `release/X.X.X` branches.                    |
-| release/X.X.X                                                                          | Accepts PRs for all changes, particularly those not backwards compatible with `develop` and `master`. |
+Please read this section if you're planning to fork this repository, or make frequent PRs into this repository.
 
-### Overview
+### Production Releases
+
+Production releases are always tags, versioned as `<component-name>/v<semver>`.
+For example, an `op-node` release might be versioned as `op-node/v1.1.2`, and an L1 smart contract release might be versioned as `contracts-L1/v1.0.0`.
```

Comment:
Leaving this PR as a draft until we have the `contracts-L1/v1.0.0` and `contracts-L2/v1.0.0` releases. 

We probably also need releases for e.g. the EAS and governance directories?

Reply:
We have had discussion on moving the contracts to the root in a package called `op-contracts` and then splitting up the packages into different forge projects to improve compile time. We just need to ensure that the naming scheme we use here matches, which we can do so by keeping the same names as the folders in `contracts-bedrock`. Thoughts on this?

Reply:
> We have had discussion on moving the contracts to the root in a package called `op-contracts`

Definitely supportive of this. In general the `op-*` vs. `packages/*` distinction feels unclear to me, and flattening feels cleaner

> and then splitting up the packages into different forge projects to improve compile time. We just need to ensure that the naming scheme we use here matches, which we can do so by keeping the same names as the folders in `contracts-bedrock`. Thoughts on this?

Can you expand on what would get split out? Also what do you mean by the naming scheme must match?

I think you are suggesting something like this:
```
op-contracts
├── cannon # forge project for cannon
├── dispute # forge project for dispute game
├── etc. 
```
If so I assume the main blocker is properly handling cross-folder dependencies, e.g. both L1 and L2 folders import from universal, so we might need to do some more refactoring.

Perhaps each folder contains unit tests from that forge project only, and integration tests live an a separate `integration` forge project?

---

Either way, I don't think these decisions/refactors should block our initial contracts release, as long as we can decide on the set of contract packages that should be released